src/kraft_controller.erl

```diff
@@ -1,16 +1,72 @@
 -module(kraft_controller).
 
+-moduledoc """
+Template-based controller handler for Kraft framework.
+
+This module provides a simplified controller for rendering HTML templates.
+It implements the `kraft_handler` behaviour and is designed for pages that need
```

Comment:
I don't think the user needs to know what behaviors the current module implements, as that is an implementation detail. I would prefer if the documentation is more user focused.